hazelcast/src/test/java/com/hazelcast/test/HazelcastTestSupport.java

```diff
@@ -1246,31 +1151,6 @@ public static void assertTrueAllTheTime(AssertTask task, long durationSeconds) {
         }
     }
 
-    public static void assertFalseEventually(AssertTask task, long timeoutSeconds) {
```

Comment:
I think that this method is used in EE repo:
https://github.com/hazelcast/hazelcast-enterprise/blob/3f54b2f0043339b923c7e9cd020f1afa7e3461e3/hazelcast-enterprise-it/hazelcast-it-k8s-tests/src/test/java/com/hazelcast/test/kubernetes/HazelcastEnterpriseHelmTestHelper.java#L15